core/modules/inputmanager.js

```diff
@@ -0,0 +1,72 @@
+/*\
+title: $:/core/modules/inputmanager.js
+type: application/javascript
+module-type: global
+
+Input handling utilities
+
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+function InputManager(options) {
+	options = options || "";
```

Comment:
Is options a string or an object? IMO it needs to be `|| {}`

Reply:
I'd add: 

```
this.wiki = options.wiki || $tw.wiki;
```

see comment below!

Reply:
I just copied this over from our keyboardManager